packages/tables/resources/lang/ka/table.php

```diff
@@ -13,22 +33,28 @@
 
     'pagination' => [
 
-        'label' => 'გვერდების ნავიგაცია',
+        'label' => 'გვერდებზე ნავიგაცია',
 
-        'overview' => 'Showing :first to :last of :total results',
+        'overview' => '{1} ნაჩვენებია ერთი შედეგი|[2,*] ნაჩვენებია :first-დან :last-მდე :total-დან',
```

Comment:
Have you tested if this works?
Seems like there's no space between :first and the word after it?

Reply:
Yes this will work with `trans_choice` function. Example:

<img width="807" alt="image" src="https://user-images.githubusercontent.com/1139050/231163960-33db3bad-d722-4910-9a83-b1225cedd22f.png">